uint8 red_low_hue
uint8 red_low_sat
uint8 red_low_val
uint8 red_high_hue
uint8 red_high_sat
uint8 red_high_val

uint8 green_low_hue
uint8 green_low_sat
uint8 green_low_val
uint8 green_high_hue
uint8 green_high_sat
uint8 green_high_val

int32 brightness
int32 contrast
uint16 roi_y
float64 gamma
